pkg/adapter/nimbus-kyverno/utils/utils.go

```diff
@@ -37,6 +40,12 @@ func GetGVK(kind string) string {
 		kind = "StatefulSet"
 	case "daemonset" :
 		kind = "DaemonSet"
+	case "configmap":
```

Comment:
Why do we need these other kinds  ? AFAIK, kyverno auto generates policies for these once we add policies for a pod

Reply:
Yes, you're right, resolved the issue, but kept the utils file intact as we may require this in future if use-case arises